### PR TITLE
Promotion script, tagging images & tagging repo

### PR DESCRIPTION
**Pull request by oshercc**
* remove tagging on the pipeline since it is not being used, commuting is good enough and no need to spam with tags
* tested, see tags "test" & "test.02.09.2020-22.47" tag cleanup will be done after this is pushed 
* after pushing the job: nightly_latest needs to be updated to run Jenkinsfile.daily_latest

### CODE REVIEW

File: tools/assisted_installer_stable_promotion.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import logging
+from datetime import datetime
+logging.basicConfig(format='%(asctime)s %(message)s')
+logging.getLogger().setLevel(logging.INFO)
+
+IMAGE_FORMAT = "quay.io/ocpmetal/{image_name}:{tag}"
+
+#######################################################################################################
+# This will promote assisted-installer after testing
+# Actions: - tag assisted installer with passed <tag> & <tag>.%d/%m/%Y-%H-%M
```
Comment (ronniel1) [inline tools/assisted_installer_stable_promotion.py:15]: "tag assisted-installer-deployment git repo"

File: tools/assisted_installer_stable_promotion.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    tag_repo(args.tag)
+    tag_repo(timestamped_tag)
+
+def tag_manifest_image():
+    with open(args.deployment, "r") as f:
+        deployment = yaml.load(f)
+    for rep, rep_data in deployment.items():
+        project, repo = rep.split("/")
+        if project != "openshift":
+            continue
```
Comment (ronniel1) [inline tools/assisted_installer_stable_promotion.py:42]: Remove this
We also tag the facet image

File: tools/assisted_installer_stable_promotion.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+timestamped_tag = "{tag}.{timestamp}".format(
+    tag=args.tag, timestamp=datetime.now().strftime("%d.%m.%Y-%H.%M"))
+
+def main():
+    tag_manifest_image()
+    tag_repo(args.tag)
+    tag_repo(timestamped_tag)
+
+def tag_manifest_image():
```
Comment (ronniel1) [inline tools/assisted_installer_stable_promotion.py:36]: tag_manifest_images

File: tools/assisted_installer_stable_promotion.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    subprocess.check_output("docker tag {} {}".format(image, timestamp_tagged_image), shell=True)
+
+    # Pushing images
+    subprocess.check_output("docker push {}".format(tagged_image), shell=True)
+    subprocess.check_output("docker push {}".format(timestamp_tagged_image), shell=True)
+
+def tag_repo(tag):
+    logging.info("Tagging repo with {tag}".format(tag=tag))
+    subprocess.check_output("git tag {tag} -f".format(tag=tag), shell=True)
+    subprocess.check_output("git push -f --tags", shell=True)
```
Comment (ronniel1) [inline tools/assisted_installer_stable_promotion.py:71]: "git push origin :{tag} -f"

Comment (oshercc) [reply tools/assisted_installer_stable_promotion.py:71]: when using this we will need to delete first the tag so we will still get the same number of lines 
```oscohen@localhost:master:~/sandbox/test_repo> git push origin :not_existing -f
error: unable to delete 'not_existing': remote ref does not exist
```

Comment (oshercc) [reply tools/assisted_installer_stable_promotion.py:68]: same issue with the second command 
```
git push origin not_existing -f
error: src refspec not_existing does not match any
error: failed to push some refs to 'https://github.com/oshercc/ignition-manifests-and-kubeconfig-generate.git'
```

**Pull request merged**